libethereum/Account.cpp

```diff
@@ -21,17 +21,22 @@
 
 #include "Account.h"
 #include <json_spirit/JsonSpiritHeaders.h>
-#include <libethcore/Common.h>
 #include <libethcore/ChainOperationParams.h>
 #include <libethcore/Precompiled.h>
+
 using namespace std;
 using namespace dev;
 using namespace dev::eth;
-namespace js = json_spirit;
 
-#pragma GCC diagnostic ignored "-Wunused-variable"
+void Account::setNewCode(bytes&& _code)
+{
+	// FIXME: Make code hash lazy (calculated when first needed)?
```

Comment:
I don't think this is necessary. We have to compute the hash unless the creation is reverted.